services/accountsansjuggler/repositories/account/datasources/mysqlds.ts

```diff
@@ -0,0 +1,180 @@
+const debug = require('debug')('accountsansjuggler');
+const mysql = require('mysql');
+import { CrudConnector, EntityData } from "loopback-next/packages/repository/src/crud-connector";
+import {
+    Class, 
+    DataSource, 
+    Entity, 
+    Filter, 
+    ObjectType, 
+    Options, 
+    Where
+} from 'loopback-next/packages/repository';
+
+export class MySqlConn implements CrudConnector {
+    private connection
+
+    constructor(config: Object) {
+      this.connection = mysql.createConnection(config);
+      if (this.connection.state !== 'authenticated') {
+        this.connect().then(function(res) {
+          debug('Connection success ', res);
+        }, function(err) {
+          debug('err connecting', err);
+    });
+}
+}
+      name: 'mysql';
+      interfaces?: string[];
+      connect() {
+          let self = this;
+          return new Promise<void>(function(resolve, reject) {
+              self.connection.connect(function(err) {
+                  if (err !== null) return reject(err);
+                  resolve(err);
+              })
+          })
+      } 
+      disconnect(){
+          let self = this;
+         return new Promise<void>(function(resolve, reject) {
+              self.connection.end(function(err) {
+                  if (err !== null) return reject(err);
+                  resolve(err);
+              })
+          })
+      }
+      ping(){
+          let self = this;
+         return new Promise<void>(function(resolve, reject) {
+              self.connection.ping(function(err) {
+                  if (err !== null) return reject(err);
+                  resolve(err);
+              })
+          })
+      }
+
```

Comment:
Please format the code with correct indentation. If you use VSCode, try `prettier javascript formatter`.